lib/pip.js

```diff
@@ -87,19 +86,17 @@ function installRequirements(requirementsPath, targetFolder, serverless, service
       cmdOptions.push('-e', 'SSH_AUTH_SOCK=/tmp/ssh_sock');
     }
     if (process.platform === 'linux') {
-      // Set the ownership of the .serverless/requirements folder to current user
-      pipCmd = quote(pipCmd);
-      const chownCmd = quote([
-        'chown', '-R', `${process.getuid()}:${process.getgid()}`,
-        targetRequirementsFolder,
-      ]);
-      pipCmd = ['/bin/bash', '-c', '"' + pipCmd + ' && ' + chownCmd + '"'];
+      // Use same user so requirements folder is not root and so --cache-dir works
+      cmdOptions.push('-u', `${process.getuid()}`);
       // const stripCmd = quote([
       //   'find', targetRequirementsFolder,
       //   '-name', '"*.so"',
       //   '-exec', 'strip', '{}', '\;',
       // ]);
       // pipCmd = ['/bin/bash', '-c', '"' + pipCmd + ' && ' + stripCmd + ' && ' + chownCmd + '"'];
+    } else if (process.platform === 'win32') {
```

Comment:
Could you change this to an empty else (or add macOS/`darwin`)? and I'll have some one with a mac test it.

Reply:
Nope, AFAIK circle doesn't offer free mac os builds.

Reply:
Would you be open to moving to Travis.CI? Mac builds are free there and I think they have a working Docker setup too.